### PR TITLE
Release 0.7.0, happy birthday Habitat, have a hart <3

### PR DESCRIPTION
**Pull request by fnichol**

### CODE REVIEW

**Comment by thesentinels** [comment]
By analyzing the blame information on this pull request, we identified @reset, @jtimberman, @metadave, @adamhjk and @smith to be potential reviewers

File: components/ruby-client/lib/habitat/client/version.rb [hunk 1]
```diff
@@ -17,6 +17,6 @@
 
 module Habitat
   class Client
-    VERSION = '0.6.0'.freeze
+    VERSION = '0.7.0'.freeze
```
Comment (metadave) [inline components/ruby-client/lib/habitat/client/version.rb]: ping @jtimberman

**Comment by metadave** [comment]
@thesentinels r+

gif-keyboard-12285104434174892776

**Comment by thesentinels** [comment]
:pushpin: Commit b8800a7 has been approved by `metadave`

**Comment by thesentinels** [comment]
:hourglass: Testing commit b8800a7 with merge c94f166...

**Comment by fnichol** [comment]
gif-keyboard-10843209565649239669

**Comment by thesentinels** [comment]
:sunny: Test successful - [travis](https://travis-ci.com/habitat-sh/habitat/builds/26838528)

**Pull request merged**